packages/quip-cli/templates/js_webpack/webpack.config.js

```diff
@@ -0,0 +1,40 @@
+const path = require("path");
+const devMode = process.env.NODE_ENV === "development";
+
+module.exports = {
+    mode: devMode ? "development" : "production",
+    entry: [path.resolve(__dirname, "./src/root.jsx")],
+    output: {
+        path: path.resolve(__dirname, "./dist"),
+        filename: "app.js",
+        publicPath: "dist",
+    },
+    resolve: {
+        modules: [
+            path.resolve(__dirname, "src"),
+            path.resolve(__dirname, "node_modules"),
+        ],
+        extensions: [".js", ".jsx"],
+    },
+    resolveLoader: {
+        modules: [path.resolve(__dirname, "node_modules")],
+    },
+    module: {
+        rules: [
+            {
+                test: /\.jsx?$/,
+                exclude: /node_modules/,
+                loader: "babel-loader",
+            },
+        ],
+    },
+    devServer: {
+        contentBase: path.resolve(__dirname),
+        port: 8888,
+        inline: false,
+    },
+    externals: {
```

Comment:
If we can we should drop the externals, since app code should work without them (via the quiptext and quip-apps-api packages) and new apps should never use the globals.